src/components/CreatePost/ComplexButton.tsx

```diff
@@ -10,43 +10,51 @@ interface ComplexButtonProps {
 }
 
 const ComplexButton = ({ value, handleChange }: ComplexButtonProps) => {
+  const handleReset = (e: MouseEvent<HTMLElement>) => {
+    handleChange(e, '');
+  };
```

Comment:
p1;

```suggestion
  const onReset = (e: MouseEvent<HTMLElement>) => {
    handleChange(e, '');
  };
```


![image](https://user-images.githubusercontent.com/57402711/223285999-205abccf-e3fe-4357-a007-11502717d3be.png)

Reply:
![image](https://user-images.githubusercontent.com/105067764/223339934-c3d42783-86fc-4b04-9a3b-544274d32f14.png)

저희 이걸로 바뀌었어요...

Reply:
handleChange를 onChange로 바꾸어야겠네요